.circleci/docker/Dockerfile.ubuntu1904

```diff
@@ -33,7 +33,7 @@ RUN set -ex; \
 	apt-get install -qqy --no-install-recommends \
 		build-essential \
 		software-properties-common \
-		cmake ninja-build clang++-8 \
+		cmake ninja-build clang++-8 libc++-8-dev \
```

Comment:
This should not harm any other build, so it should be fine - @christianparpart @Marenz can one of you update the docker image like this, so that we can see if the other change in this PR works with this change to the image?

Reply:
@christianparpart seems to have started with it.

Reply:
thank you for the update :fireworks:

Reply:
The update is optional now since I changed back to linking against `libstd++`

Reply:
I pushed an updated image to docker hub that now contains `libc++-8-dev` and `libc++abi-8-dev` although they are only meant to be used when a PR for #7257 is ready.